x-pack/test/functional/apps/ml/data_frame_analytics/regression_creation.ts

```diff
@@ -345,6 +372,44 @@ export default function ({ getService }: FtrProviderContext) {
           );
         });
 
+        it('adds discover custom url to the analytics job', async () => {
+          await ml.testExecution.logTestStep('opens edit flyout for discover url');
+          await ml.dataFrameAnalyticsTable.openEditFlyout(testData.jobId);
+          await ml.dataFrameAnalyticsEdit.openEditCustomUrlsForJobTab(testData.jobId);
+
+          await ml.testExecution.logTestStep('add discover custom url for the analytics job');
+          await ml.dataFrameAnalyticsEdit.addDiscoverCustomUrl(testDiscoverCustomUrl);
+
+          await ml.dataFrameAnalyticsEdit.updateAnalyticsJob();
+        });
+
+        it('adds dashboard custom urls to the analytics job', async () => {
```

Comment:
Nit - `adds dashboard custom url`

Reply:
Removed in 518fabb4bc2b59e729c93231c9c258c0cbffa1b3